# Exploring RNA-Seq Data: FACT in Genome Architecture
## Introduction
This README provides a general overview of the exploration of RNA-Seq data from the paper titled "FACT mediates cohesin function on chromatin". This study investigates the relationship between the cohesin complex and histone chaperon 'facilitates chromatin transcription' (FACT) in Saccharomyces cerevisiae, shedding light on their roles in chromosome organization and architecture. 

## Paper Overview
Title: FACT mediates cohesin function on chromatin

Authors: Jonay Garcia-Luis, Luciana Lazar-Stefanita, Pilar Gutierrez-Escribano, Agnes Thierry, Axel Cournac, Alicia García, Sara González, Mar Sánchez, Adam Jarmuz, Alex Montoya, Marian Dore, Holger Kramer, Mohammad M. Karimi, Francisco Antequera, Romain Koszul & Luis Aragon

Published in: Nature Structural & Molecular Biology, 2019

Link to Paper: https://www.nature.com/articles/s41594-019-0307-x

## Research Background
Cohesin is a protein complex with essential roles in maintaining genome stability. It is particularly well known for its functions in sister chromatid cohesion and its involvement in the organization of chromosomes. 

In this study, the researchers explored the histone chaperone FACT's effect on cohesin. The paper reveals that FACT is critical for cohesin's role in chromosome organization.

## RNA-Seq Data Exploration
The exploration of RNA-Seq data in this study is fundamental to understanding how changes in gene expression are correlated with the observed effects on chromosome structure and function. The RNA-Seq data provides insight into the changes associated with the depletion of FACT and its impact on the gene environment. 

We will explore this data to produce our own findings on how a depletion of FACT affects various pathways. The analysis involves the following steps:

1. Data Collection: We will retrieve RNA-Seq data from archives.
2. Data Preprocessing: We will clean and preprocess the raw RNA-Seq data, which includes quality control, adapter trimming, and read alignment to the reference genome.
3. Differential Gene Expression Analysis: We will compare gene expression profiles between control and FACT-depleted cells to identify significant differentially expressed genes.
4. Enrichment Analysis: We will explore the biological pathways and gene ontology terms to understand the functional consequences of FACT depletion.

For a thorough and detailed understanding of the study's results and methodology, please refer to the original paper. 
